README.md

```diff
@@ -79,6 +79,24 @@ Default region name [None]: eu-central-1
 Default output format [None]: json
 ```
 
+If you have been provided with session token, you can add it via following command:

```

Comment:
There a various ways how to authenticate: https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication-and-configuration
What you need to run terraform depends on your specific setup. This should be reflected in the documentation.